packages/vue2-jest/lib/process.js

```diff
@@ -8,8 +8,18 @@ const stripInlineSourceMap = require('./utils').stripInlineSourceMap
 const getCustomTransformer = require('./utils').getCustomTransformer
 const loadSrc = require('./utils').loadSrc
 const babelTransformer = require('babel-jest').default
-const compilerUtils = require('@vue/component-compiler-utils')
```

Comment:
@thebanjomatic this import used to be here anyway, without being declared explicitly in the peer dependencies. My exposure to this codebase is limited so I am not really aware of the consequences of these changes given the amount of permutations we can have in regards of `vue-jest`,`jest`, and `vue` versions.

Reply:
The thing is that its already a peer dependency, the project just isn't declaring it in the package.json manifest. This would have issues if you were using yarn pnp for example because it is trying to import a package that isn't declared and you would need to use packageExtensions to inject those declarations as consumers of the library.

If you don't want to make the change in this PR, that's fine. I might pursue a separate PR later to clean up these undeclared peerDependencies and to fully optionalize vue-template-compiler so that you don't need to install that separately for vue 2.7 also.

I do think `vue/compiler-sfc` is still the better import than `@vue/compiler-sfc` though since we already have the vue peer dependency declared and this appears to be the preferred way to import compiler-sfc now that it ships with the vue dependency out of the box.

Reply:
@thebanjomatic that’s a fair point. I can change this PR. Will update it later.

Reply:
LMK when it's ready for a re-review.

Providing compat for all versions is really hard and tedious... :|

Reply:
PS just released v29 for Jest 29

You might need to rebase. If this merges, the changes will be in v29 - we can back port to v28, but it's a lot of work to maintain multiple versions, I generally ask people to update to the latest Jest.

 https://github.com/vuejs/vue-jest/releases/tag/v29.0.0

Reply:
@lmiller1990 sorry I’ve been kinda busy last week, so what’s your recommendation? Make this change only compatible with Jest 29?